### PR TITLE
German cards

### PR DESCRIPTION
**Pull request by juliagolemo**
Issue: #37

### CODE REVIEW

File: src/App.js
Comment (SzymonCogiel) [inline src/App.js:1]: Co to ma być🤯

File: public/images/markdown/german.jpg
Comment (juliagolemo) [inline public/images/markdown/german.jpg:1]: Remove this file, it is redundant.

Comment (SzymonCogiel) [inline public/images/markdown/german.jpg:1]: Remove this file, it is redundant

**Review by SzymonCogiel — APPROVED** [review]
Good piece of work!

**Pull request merged**